src/components/Grid/Col.md

```diff
@@ -123,3 +123,23 @@ The `last` property can pull a column to the right
 </Container>
 ```
 <!-- prettier-ignore-end -->
+
+### Set elements of containers
+
```

Comment:
It would be worthwhile to have additional documentation about why users would want to do this and what it means to set the `component` prop.